components/core/tools/docker-images/clp-env-base-musllinux_1_2-aarch64/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+component_root="${script_dir}/../../../"
+
+build_cmd=(
+    docker buildx build
+    --platform linux/arm64
+    --tag clp-core-dependencies-aarch64-musllinux_1_2:dev
+    --file "${script_dir}/Dockerfile"
+    --load
+    "$component_root"
```

Comment:
```suggestion
    --file "${script_dir}/Dockerfile"
    --load
    "$component_root"
```
```suggestion
    "$component_root"
    --file "${script_dir}/Dockerfile"
    --load
```
To be consistent with other image's build script.